posts/hello-world-in-kafka-using-python.md

```diff
@@ -19,30 +19,36 @@ You might have got a hint of it already. This blog is about understanding what i
 ## What is Kafka? Why should one use it?
 In short, Kafka is a distributed streaming and message queueing platform.
 
-> Oh wait! What does that even mean? How different is it from traditional databases? 
+> Oh wait! What does that even mean?
 
+A streaming platform is a system that can perform the following:
+1. Store and process streams of data in real time
```

Comment:
What do you mean by streams of data? A beginner will not have an idea about streams. Consider a full stack dev, what does stream means for a full stack dev.